cmd/snap/main.go

```diff
@@ -174,9 +174,15 @@ enabling secure distribution of the latest apps and utilities for
 cloud, servers, desktops and the internet of things.
 
 This is the CLI for snapd, a background service that takes care of
-snaps on the system. Start with 'snap list' to see installed snaps.
-`)
-	parser.FindOptionByLongName("version").Description = i18n.G("Print the version and exit")
+snaps on the system. Start with 'snap list' to see installed snaps.`)
```

Comment:
newline?

Reply:
No, I removed that one on purpose (look at the output)